test/clone/unit/index.js

```diff
@@ -0,0 +1,137 @@
+import { assert } from 'chai';
+import clone from '../../../clone';
+
+describe('Clone', () => {
+
+    it('should handle null', () => {
+        assert.isNull(clone(null));
+    });
+
+    it('should handle undefined', () => {
+        assert.isUndefined(clone(undefined));
+    });
+
+    it('should handle booleans', () => {
+        assert.isFalse(clone(false));
+        assert.isTrue(clone(true));
+    });
+
+    it('should handle numbers', () => {
+        assert.strictEqual(clone(42), 42);
+        assert.strictEqual(clone(3.14159265358979), 3.14159265358979);
+        assert.strictEqual(clone(0), 0);
+        assert.strictEqual(clone(-1.6180339887), -1.6180339887);
+        assert.strictEqual(clone(Number.POSITIVE_INFINITY), Number.POSITIVE_INFINITY);
+        assert.strictEqual(clone(Number.NEGATIVE_INFINITY), Number.NEGATIVE_INFINITY);
+    });
+
+    it('should handle strings', () => {
+        assert.strictEqual(clone('Hello, World!'), 'Hello, World!');
```

Comment:
That's an interesting decision to not follow the historical precedence of "hello, world".